plugin/content-script.js

```diff
@@ -749,21 +714,30 @@ function onUrlChange() {
 		waitForElement("[class^='ButtonHoverWrapper']", 10000)
 		.then( () => {
 			createOffersHelperContainer();
+				watching4histories = setInterval(function() {
+					waitForElement(".HistoryItemWrapper-sc-13gqei4-0", 1000)
+					.then( () => {
+						let splitted_url = lastUrl.split('/');
+						fetch(`https://api.nft.gamestop.com/nft-svc-marketplace/getNft?tokenIdAndContractAddress=${splitted_url[5]}_${splitted_url[4]}`)
```

Comment:
Let's break this out into it's own parameterized `function` with callback, so that we can reference it for other API calls in the future.

Reply:
Do it

Reply:
Done.